src/internal/m365/collection/drive/delta_tree_test.go

```diff
@@ -651,3 +654,202 @@ func (suite *DeltaTreeUnitSuite) TestFolderyMcFolderFace_SetFolder_correctTombst
 
 	entomb().compare(t, tree.tombstones)
 }
+
+// ---------------------------------------------------------------------------
+// file tests
+// ---------------------------------------------------------------------------
+
+func (suite *DeltaTreeUnitSuite) TestFolderyMcFolderFace_AddFile() {
```

Comment:
could you please either expand this set of tests or add a new test function that checks that setting the mod time works right? I think the cases that really need to be tested are adding a new file and updating/moving a file

Reply:
will revist later, since I'll be swapping out the time for the full drive item in an upcoming PR.

Reply:
can we make a list of all the "we'll do this later" things please? We seem to be gathering quite a few loose ends and I'd like to make sure they get addressed